certbot/cert_manager.py

```diff
@@ -121,7 +121,27 @@ def domains_for_certname(config, certname):
     return lineage.names() if lineage else None
 
 def find_duplicative_certs(config, domains):
-    """Find existing certs that duplicate the request."""
+    """Find existing certs that match the given domain names.
+
+    This function searches for certificates whose domains are equal to
```

Comment:
Let's put \`s around `domains` to make it clear when we mean the parameter name.